website/src/config.ts

```diff
@@ -183,17 +183,48 @@ function getAccessionInputField(): InputField {
     };
 }
 
-function getSubmissionIdInputField(): InputField {
-    return {
-        name: SUBMISSION_ID_INPUT_FIELD,
-        displayName: 'ID',
-        definition: 'FASTA ID',
-        guidance:
-            'Your sequence identifier; should match the FASTA file header - this is used to link the metadata to the FASTA sequence',
-        example: 'GJP123',
-        noEdit: true,
-        required: true,
-    };
+export function getSubmissionIdInputFields(isMultiSegmented: boolean): InputField[] {
+    if (!isMultiSegmented) {
+        return [
+            {
+                name: SUBMISSION_ID_INPUT_FIELD,
+                displayName: 'ID',
+                definition: 'FASTA ID',
+                guidance:
+                    'Your sequence identifier; should match the FASTA file header - this is used to link the metadata to the FASTA sequence.',
+                example: 'GJP123',
+                noEdit: true,
+                required: true,
+            },
+        ];
+    }
+    return [
+        {
+            name: SUBMISSION_ID_INPUT_FIELD,
+            displayName: 'ID',
+            definition: 'METADATA ID',
+            guidance:
+                'Your sample identifier. If no FastaId is provided, this ID will be used to associate the metadata with the sequence.',
```

Comment:
Personally my suggestion would be to leave out the last sentence, and if we describe this describe it in text elsewhere. (And in theory if we include it we should make it `fastaIds` or something.)

But this isn't blocking.